doc/lightning-listconfigs.7.md

```diff
@@ -88,7 +88,8 @@ On success, an object is returned, containing:
 - **autolisten** (boolean, optional): `autolisten` field from config or cmdline, or default
 - **proxy** (string, optional): `proxy` field from config or cmdline, or default
 - **disable-dns** (boolean, optional): `true` if `disable-dns` was set in config or cmdline
-- **disable-ip-discovery** (boolean, optional): `true` if `disable-ip-discovery` was set in config or cmdline
+- **disable-ip-discovery** (boolean, optional): `true` if `disable-ip-discovery` was set in config or cmdline (DEPRECATED)
+- **ip-discovery** (string, optional): `true` if `ip-discovery` was set in config or cmdline
```

Comment:
Perhaps `use-discovered-ip-addresses`.  Then true/false/auto works.  And this comment surely is wrong?

Reply:
Change it to `announce-addr-discovered` to be inline with all the `announce-addr...` config options.